### ISSUE TITLE
icons not showing

### ISSUE THREAD

**Issue by frankleng** [Dogfalo/materialize#1079]
is it just me, or are icons are not working for me after the 0.96 update.?

**Comment by dhawken** [comment]
Just updated and mine seem fine.

**Comment by sentenza80** [comment]
It's working here

**Comment by jaimevent** [comment]
I had the same issue.
Solved it replacing at line ~781 of the file sass/components/_icons_material-design.scss the following:
instead of `content: unicode($mdi-icon-value);` write `content: "\""+ $mdi-icon-value) +"\""`

**Comment by frankleng** [comment]
@jaimevent thanks! that totally fixed it

**Comment by frankleng** [comment]
@Dogfalo why is the line @jaimevent referenced to not in
https://github.com/Dogfalo/materialize/blob/master/sass/components/_icons-material-design.scss

want to do a PR to fix this issue, but not sure where to start.

**Comment by Dogfalo** [comment]
We updated the icon font

**Comment by frankleng** [comment]
@Dogfalo thanks for the reply.
I'm using gulp-sass which pulls in node-sass 3.0.0-beta.4

I can build the latest materialize.scss and icons are showing. but they are all mismatched.

**Comment by Dogfalo** [comment]
You need to grab the updated don't files as well

**Comment by frankleng** [comment]
ah of course! found a bug in my build script that didn't clean up the font.
Thanks!!!

**Comment by cascalheira** [comment]
I had the same problem with the current repository code. @jaimevent solution worked for me.
Using broccoli-sass on ember-cli.

**Comment by Chun-Yang** [comment]
@jaimevent solved the bug.

**Comment by lsunsi** [comment]
I'm using the Meteor version of Materialize and I'm icons are not showing up, only the icon text. Am I missing something?

**Comment by carlosperate** [comment]
http://materializecss.com/icons.html

**Comment by lsunsi** [comment]
Entirely my bad.
Thanks

**Comment by thg303** [comment]
there's a problem with "Material-Design-Icons.woff" in the package, I've replaced it with the google version below, and it worked.
https://fonts.gstatic.com/s/materialicons/v7/2fcrYFNaTjcS6g4U3t-Y5ewrjPiaoEww8AihgqWRJAo.woff

**Comment by nitesh-bhargav** [comment]
@thg303 Thanks, it worked for me. @Dogfalo  you need to update this in your V0.97 package.

**Comment by Dogfalo** [comment]
![](http://puu.sh/jVTCb/6ddaf609d3.png)

**Comment by nitesh-bhargav** [comment]
@Dogfalo unfortunately the problem in firefox is not solved by adding the icon link, we have to change the .woff file. Well the problem occurred after the .97 update.

**Comment by ztvMark** [comment]
SOLUTION
@font-face {
  font-family: 'Material Icons';
  font-style: normal;
  font-weight: 400;
  src: local('Material Icons'), local('MaterialIcons-Regular'), url(http://fonts.gstatic.com/s/materialicons/v7/2fcrYFNaTjcS6g4U3t-Y5ZjZjT5FdEJ140U2DJYC3mY.woff2) format('woff2');
}

.material-icons {
  font-family: 'Material Icons';
  font-weight: normal;
  font-style: normal;
  font-size: 24px;
  line-height: 1;
  letter-spacing: normal;
  text-transform: none;
  display: inline-block;
  word-wrap: normal;
  -webkit-font-feature-settings: 'liga';
  -webkit-font-smoothing: antialiased;
}

**Comment by andregoldstein** [comment]
@ztvMark :+1:

**Comment by b-z** [comment]
@ztvMark 
I just downloaded the .woff2 file and it works!
Gracias!

**Comment by replay111** [comment]
I have just created project with gulp-sass, added with bower install materialize --save Materialize and copied from bower_components/Materialize/font to <my_project>/src/font with directory structure.
During gulp serve I can't see that browser is loading anything with Material Icons - only roboto is loaded.
I did not change any of material scripts... why this is not working?

<img width="762" alt="zrzut ekranu 2016-01-03 o 19 17 05" src="https://cloud.githubusercontent.com/assets/5937475/12080047/a3a80aa8-b24e-11e5-9c66-3f474850b85f.png">
